integrations/azure-devops/azure_devops/webhooks/listeners/push.py

```diff
@@ -0,0 +1,53 @@
+import typing
+from typing import Any
+from loguru import logger
+from port_ocean.context.event import event
+from port_ocean.context.ocean import ocean
+from port_ocean.clients.port.types import UserAgentType
+from azure_devops.webhooks.webhook_event import WebhookEvent
+from .listener import HookListener
+from azure_devops.gitops.port_app_config import GitPortAppConfig
+from azure_devops.gitops.generate_entities import generate_entities_from_commit_id
+
+
+class PushHookListener(HookListener):
+    webhook_events = [WebhookEvent(publisherId="tfs", eventType="git.push")]
+
+    async def on_hook(self, data: dict[str, Any]) -> None:
+        logger.info("Got Push event!")
```

Comment:
try to be more explicit
maybe a generic log in the one that is calling the on_hook that will also log the event type